### PR TITLE
Oshift migration tool

### PR DESCRIPTION
**Pull request by koneill94**
Article outlining how customers could use velero as a backup/restore/migration tool in OpenShift

### CODE REVIEW

File: articles/openshift/oshift-ref-migration.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+- apiVersion: velero.io/v1
+  kind: BackupStorageLocation
+  metadata:
+    creationTimestamp: null
+    labels:
+      component: velero
+    name: default
+    namespace: velero
+  spec:
+    config: {"s3Url": "https://cas.cor00005.ukcloud.com","region": "us-east-1"}
```
Comment (gellner) [inline articles/openshift/oshift-ref-migration.md]: I know it's a required and irrelevant parameter but wouldn't `eu-west-2` be preferable, if we needed a valid AWS region? ;-)

Comment (gellner) [reply articles/openshift/oshift-ref-migration.md]: Apparently, that is how cas.cor is configured?!

```s3cmd info s3://branding-files
s3://branding-files/ (bucket):
   Location:  us-east-1
   Payer:     none
   Expiration Rule: none
   policy:    none
   cors:      none
   ACL:       dfdfddfdfgdggddff: FULL_CONTROL```

**Comment by koneill94** [comment]
> I've edit this a bit and reformatted it a bit to add numbers to the steps to make it easier to follow. I'd suggest that you read it through to make sure I haven't inadvertently messed anything up.
> 
> As it provides instructions for how to perform various tasks, it should really be a How to article rather than a Reference article. Could you change the file name and title accordingly?

Changes look good @suehighmoor thanks. I've updated the file name and title as suggested.

File: articles/openshift/oshift-how-backup-migrate.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+- Restore a namespace to a new cluster
+
+Velero offers a tool called restic that enables backup of persistent volume data regardless of provider, however this guide doesn't cover restic as the tool is considered beta-quality. If you'd like to test it you can find more on restic here: <https://velero.io/docs/v1.1.0/restic/>.
+
+### Installing Velero
+
+1. Download Velero from <https://github.com/vmware-tanzu/velero/releases>.
+
+    You can deploy Velero onto OpenShift with the `velero install` command. To deploy Velero, you'll need compatible object storage: <https://velero.io/docs/v1.1.0/support-matrix/>. In this article we use UKCloud's Cloud Storage product (based on Dell EMC's ECS).
```
Comment (stevemul) [inline articles/openshift/oshift-how-backup-migrate.md]:
```suggestion
    You can deploy Velero onto OpenShift with the `velero install` command. To deploy Velero, you'll need compatible object storage: <https://velero.io/docs/v1.1.0/support-matrix/>. In this article we use UKCloud's Cloud Storage product.
```
I'd remove the specifics about the service we use as it's not really relevant and could change in future.

**Comment by koneill94** [comment]
Commited Steve Ms suggested changes @suehighmoor think this one is good to merge

**Pull request merged**